packages/world/src/Constants.sol

```diff
@@ -40,9 +40,14 @@ uint128 constant BUILD_ENERGY_COST = 255500000000000;
 uint128 constant TILL_ENERGY_COST = 255500000000000;
 uint128 constant CRAFT_ENERGY_COST = 255500000000000;
 uint128 constant MOVE_ENERGY_COST = 25550000000000;
-uint128 constant WATER_MOVE_ENERGY_COST = MOVE_ENERGY_COST * 3;
-uint128 constant LAVA_MOVE_ENERGY_COST = MOVE_ENERGY_COST * 10;
-uint128 constant PLAYER_FALL_ENERGY_COST = MAX_PLAYER_ENERGY / 25; // This makes it so, with full energy, you die from a 25 block fall
+uint128 constant WATER_MOVE_ENERGY_COST = MAX_PLAYER_ENERGY / 4000; // 4000 moves in water to die
+uint128 constant LAVA_MOVE_ENERGY_COST = MAX_PLAYER_ENERGY / 10; // 10 moves on lava to die
+uint128 constant PLAYER_FALL_ENERGY_COST = MAX_PLAYER_ENERGY / 25; // This makes it so, with full energy, you die from a 25 + 3 block fall
```

Comment:
could we make this 25 by just setting it to 22?

the other energy cost + drain rates lgtm